Documentation

gradyw opened the pull request.

gradyw committed:
Removed unnecessary outdated config code and copied some documentation from CANSparkMax

samuelili reviewed (dismissed):
This looks great! Is there anywhere else we should likely document?

gradyw committed:
More documentation done in looper, subsystems, controls, and logger

gradyw committed:
cleanup/remove old annotation stuff

gradyw committed:
Revert "cleanup/remove old annotation stuff"

This reverts commit cb17c7855005eaa3abcbec8323fa78fedd2cd1ec.

gradyw committed:
More documentation/cleanup

1) Encapsulated Axes and Buttons in SubsystemControlsConfig.

2) Added Javadocs in Axis and Button

3) Constructors for Axis and Button now take a Controller rather than a Joystick

gradyw committed:
Solenoid group

gradyw committed:
Update CANSparkMaxWrapper.java

Javadocing

gradyw committed:
Remove all annotation @interfaces

gradyw committed:
Merge branch 'master' into documentation
